Gemfile.lock

```diff
@@ -321,6 +323,7 @@ DEPENDENCIES
   debug
   devise
   dotenv-rails
+  faker
```

Comment:
We have duplicate faker entries/versions. Not sure if this will cause a conflict or not.